kgforge/core/forge.py

```diff
@@ -737,6 +737,7 @@ def update(
 
         :param data: the resources to update
         :param schema_id: an identifier of the schema the updated resources should conform to
+        :param update_schema: whether the resource(s)' schemas should be updated to the schema_id value
```

Comment:
I don't see `update_schema` among the function arguments.

Reply:
My bad, initially I thought that both schema_id and a flag should be provided to enable schema change, but then it turns out only schema_id was necessary. I had made an implementation with the update_schema boolean, but then removed it and forgot to remove the docstring